assets/scss/00-base/mixins/_ma-typography.scss

```diff
@@ -2,41 +2,33 @@
   font-size: 3rem;
   line-height: 1.07;
 
-  @media ($bp-medium-min){
+  @media ($bp-medium-min) {
     font-size: 3.25rem;
   }
 
-  @media ($bp-large-min){
-    font-size: 4rem;
-  }
-
-  @media ($bp-x-large-min){
-    font-size: 4.375rem;
+  @media ($bp-x-large-min) {
+    font-size: 3.5rem;
   }
 }
 
 @mixin ma-h2 {
-  font-size: 2.25rem;
-  line-height: 1.222222;
-
-  @media ($bp-small-min){
-    font-size: 2.688rem;
-  }
+  font-size: 2rem;
```

Comment:
why are we reducing the font-size of h2 while bumping up h3 here --> seems counterintuitive to have h2 < h3